### DISCUSSION TITLE
part-ii/2017-115/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#43] [Question comments]
Part II 2017 Principles of Quantum Mechanics: Paper 4, Section II, C 

https://questions.tripos.org/part-ii/2017-115/

**Comment by silkca** [comment]
This is a standard calculation - the first-order term in the state as it evolves in time dependent perturbation theory is known, and the second-order term in the probability is naturally its modulus squared.

The second part simply requires you to substitute the relevant values into the expression for second-order probability and perform the integral.
